app/services/closure_decision_tree.rb

```diff
@@ -10,7 +10,9 @@ def destination
     when :additional_info
       edit(:support_documents)
     when :support_documents
-      show(:start) # TODO: check your answers step
+      show(:check_answers)
+    when :check_answers
+      task_list
```

Comment:
Probably to the home? the page that originated the journey (closure in this case). At least for now, we can always change it easily if we decide other option. We need to also update the appeal flow as it is also using task_list